packages/block-editor/src/components/inserter/search-results.js

```diff
@@ -88,8 +89,39 @@ function InserterSearchResults( {
 		if ( maxBlockTypesToShow === 0 ) {
 			return [];
 		}
+
+		// Force first block to be the one that is shown first
+		const reversedforceBlockInserterItems =
+			forceBlockInserterItems?.reverse();
+
+		/**
+		 * Allow for consumer to overide the default sorting of the block types.
+		 * This is useful for when you want to force a block(s) to be shown first.
+		 * The remaining blocks will obey the "frecency" sorting.
+		 *
+		 * @param {Object} resultItem a block type item result to be sorted.
+		 * @return {number} the sorting value.
+		 */
+		const comparatorField = reversedforceBlockInserterItems?.length
+			? ( resultItem ) => {
+					if (
+						reversedforceBlockInserterItems.includes(
+							resultItem.id
+						)
+					) {
+						return (
+							10000 +
```

Comment:
This constant value needs to be higher than any `frecency` value attached to a given block. I do worry that for very active sites `core/paragragh` could be very high.